minikube/bootstrap.sh

```diff
@@ -91,6 +100,78 @@ upgradeHomebrewPackages() {
   done
 }
 
+# upgradeChocoPackages upgrades required Chocolatey packages to latest version.
+upgradeChocoPackages() {
+  runIfNot "choco list --local-only | grep minikube" \
+    choco install --yes minikube
+  if [ "$(choco outdated | grep minikube)" ]; then
+    runCmd \
+      choco upgrade --yes minikube
+  fi
+  for pkg in kubernetes-cli kubernetes-helm; do
+    if [ ! "$(choco list --local-only | grep $pkg)" ]; then
+      runCmd \
+        choco install --yes $pkg
+    elif [ "$(choco outdated | grep $pkg)" ]; then
+      runCmd \
+        choco upgrade --yes $pkg
+    fi
+  done
+}
+
+# upgradeBinaryPackages upgrades required binary packages to latest version.
+upgradeBinaryPackages() {
+  if [ -z $(which helm) ]; then
+    installHelmBinary
+  else
+    HELM_LATEST_VERSION=$(getGitHubLatestRelease "helm/helm")
+    HELM_CURRENT_VERSION=$(helm version --client --short | tr "+" " " | awk '{ print $2 }')
```

Comment:
@targs08 use `local` bash variables here. Name can be shortened to `LAST_VER` and `CURR_VER`.